app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -150,7 +149,7 @@ public void addBuildingMarker(GoogleMap map, LatLng centerPos, String buildingLa
                         .anchor(0.5f,0.5f)
                         .alpha(0.90f)
                         //This line should be included whenever we test the UI for the marker:
-                        //.title(buildingLabel)
+                        .title(buildingLabel)
```

Comment:
Comment out this line here because otherwise a marker with the building label will show up everytime we click on the label

Reply:
done